crates/bevy_ecs/src/query/access.rs

```diff
@@ -469,6 +494,20 @@ impl<T: SparseSetIndex> FilteredAccess<T> {
     pub fn is_subset(&self, other: &FilteredAccess<T>) -> bool {
         self.required.is_subset(&other.required) && self.access().is_subset(other.access())
     }
+
+    /// Returns the components this access filters for.
```

Comment:
Naming and docs could be clearer here and below.

Reply:
Let me know if it's still not clear enough